dumper/app/src/main/java/com/google/edwmigration/dumper/application/dumper/ConnectorArguments.java

```diff
@@ -138,7 +138,8 @@ public class ConnectorArguments extends DefaultArguments {
     private final OptionSpec<String> optionOracleSID = parser.accepts(OPT_ORACLE_SID, "SID name for oracle").withRequiredArg().describedAs("orcl").ofType(String.class);
     private final OptionSpec<String> optionConfiguration = parser.accepts("config", "Configuration for DB connector").withRequiredArg().ofType(String.class).withValuesSeparatedBy(';').describedAs("key=val;key1=val1");
     // private final OptionSpec<String> optionDatabase = parser.accepts("database", "database (can be repeated; all if not specified)").withRequiredArg().describedAs("my_dbname").withValuesSeparatedBy(',');
-    private final OptionSpec<File> optionOutput = parser.accepts("output", "Output file").withRequiredArg().ofType(File.class).describedAs("cw-dump.zip");
+    private final OptionSpec<File> optionOutput = parser.accepts("output", "Output file (cannot be used together with --output-dir)").withRequiredArg().ofType(File.class).describedAs("cw-dump.zip");
+    private final OptionSpec<String> optionOutputDir = parser.accepts("output-dir", "Output directory where the zip file with the default filename should be created (cannot be used together with --output option). If you want to specify the filename, use --output option instead.").withRequiredArg().ofType(String.class).defaultsTo("").describedAs("/home/user/my-dump-dir");
```

Comment:
Should this be `ofType(File.class)` like `--output` is?

Reply:
We are assuming this is the folder name rather than a particular file. `String` seemed more appropriate in that case. Would love to know your thoughts on it

Reply:
My opinion is that the documentation should not be misleading to the customer. `File` is better from the implementation perspective (because it can represent both a file and a directory). However, displaying such an implementation detail in the output of the `--help` command is in my opinion confusing.

The best solution would be to use `File` here, but then output either `Directory` or (better) nothing in the `--help` command output. I think we can prepare such an improvement in a separate PR.